app/src/renderer/components/common/TmField.vue

```diff
@@ -159,7 +159,7 @@ export default {
       let formattedValue = value
 
       if (this.type === `number`) {
-        formattedValue = Number(value)
+        formattedValue = value.trim()
```

Comment:
this seems odd, trim doesn't work on numbers

Reply:
value is always a string as it comes from `$event.target.value`.